packages/x-data-grid-premium/src/tests/rowGrouping.DataGridPremium.test.tsx

```diff
@@ -2891,29 +2892,31 @@ describe('<DataGridPremium /> - Row grouping', () => {
       />,
     );
 
-    act(() => apiRef.current?.updateRows([{ id: 1, group: 'A', username: 'username 2' }]));
+    await act(() => {
```

Comment:
```suggestion
    await act(async () => {
```

`act` works synchronously if it doesn't receive a promise

Reply:
Thanks for spotting this, fixed. 👌